src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import print_function
+import argparse
+import logging
+import os
+from textwrap import dedent
+from ceph_volume import process, conf, decorators, terminal, __release__, configuration
+from ceph_volume.util import system, disk
+from ceph_volume.util import prepare as prepare_utils
+from ceph_volume.util import encryption as encryption_utils
+from ceph_volume.systemd import systemctl
+from ceph_volume.api import lvm as api
+from .list import direct_report
+
+
+logger = logging.getLogger(__name__)
+
+def activate_bluestore(meta, tmpfs, systemd):
+    # find the osd
+    osd_id = meta['osd_id']
```

Comment:
What about only activating the device this way:

`ceph-volume raw activate --device "$DEVICE" --no-systemd --no-tmpfs`.

I do this in https://github.com/rook/rook/pull/4435 and it works well.